sdk/security/keyvault/azadmin/internal/generate/transforms.go

```diff
@@ -46,6 +46,7 @@ func main() {
 	regexReplace("backup/models_serde.go", `(?:\/\/.*\s)+func \(\w \*?Error\).*\{\s(?:.+\s)+\}\s`, "")
 
 	//  modify Restore to use implementation with custom poller handler
+	regexReplace("backup/client.go", `\[PreFullRestoreResponse\], error\) \{\s(?:.+\s)+\}`, "[PreFullRestoreResponse], error) {return client.beginPreFullRestore(ctx, preRestoreOperationParameters, options)}")
```

Comment:
🤔  for my education/memory, why does pre/restore and not pre/backup need a custom poller?

Reply:
Don't know why, but the service team isn't following any standard LRO policy for restore. So I need a custom poller to match the wacky things the service is doing.

Reply:
Restore is an async PUT poller, so there should be a final GET on the original resource URL. The service is not doing the final GET, so need to use a custom poller that doesn't implement the final GET